website_breadcrumb/views/templates.xml

```diff
@@ -6,6 +6,13 @@
 <data>
 
 <template id="breadcrumb" name="Breadcrumb snippet">
+    <t t-call="website_breadcrumb.breadcrumb-generator">
+        <t t-set='include_self' t-value='False'/>
+        <t t-set='revert_order' t-value='True'/>
+    </t>
+</template>
+
+<template id="breadcrumb-generator" name="Breadcrumb snippet generator">
```

Comment:
The usual standard is to use `_` instead of `-` in ids.

Also I think that with this change, *website_breadcrumb* should depend on *website_snippet_breadcrumb* and not the other way around. If you want automatic breadcrumbs in the top of your pages, you will not care about the ability to put them manually or not; but if you want to put them manually, you will be disturbed by automatic breadcrumbs appearing in many pages.